tests/unit_tests/test_download_run_level.py

```diff
@@ -103,7 +103,7 @@ def __init__(self):
         self.label = "TheAcquisitionLabel"
 
 
-def test_download_bids_for_runlevel_acquisition_works(tmp_path, caplog):
+def test_download_bids_for_runlevel_acquisition_works(tmp_path, search_caplog, caplog):
```

Comment:
I am looking into and stealing this method.

Reply:
Maggie gave me a hard time about doing this sort of thing and I think she's generally right.  It's better to test something directly rather than testing if the log has text describing that something.  But it does work.